cdc/tests/integration_tests/_utils/check_count

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# parameter 1: expected count
+# parameter 2: component name
+# parameter 3: pd addr
+# parameter 4: max retry
+
```

Comment:
Suggest to add `set -euo pipeline`.

Reply:
I add `set -eu` instead of `set -o pipeline` ,  because `set -o pipeline`  maybe cause test to exit.